org.lflang/src/org/lflang/cli/CliBase.java

```diff
@@ -41,34 +45,61 @@
  * @author Marten Lohstroh
  * @author Christian Menard
  * @author Billy Bao
+ * @author Atharva Patil
  */
-public abstract class CliBase {
+public abstract class CliBase implements Runnable {
+    /**
+     * Models a command specification, including the options, positional
+     * parameters and subcommands supported by the command.
+     */
+    @Spec CommandSpec spec;
 
+    /**
+     * Options and parameters present in both Lfc and Lff.
+     */
+    @Parameters(
+        arity = "1..",
+        paramLabel = "FILES",
+        description = "Paths of the files to run the formatter on.")
```

Comment:
As I understand, the FILES parameter is inherited by all cli tools. Thus, the description should be more generic.

Reply:
Just fixed it.